src/main/java/org/apache/cassandra/sidecar/routes/CassandraSettingsService.java

```diff
@@ -0,0 +1,66 @@
+package org.apache.cassandra.sidecar.routes;
```

Comment:
can we add the license header to this file?

Reply:
Added the license information.